packages/core/src/Hide.js

```diff
@@ -46,5 +46,6 @@ Hide.propTypes = {
 }
 
 Hide.displayName = 'Hide'
+Hide.text = Hide.withComponent('text')
```

Comment:
FYI `withComponent` is a candidate for deprecation in styled-components v4.

Reply:
`withComponent('text')` is trying to use an html5 text tag, which isn't valid.  Maybe this could use `withComponent('span')` instead?